src/components/JsonEditorContainer/JsonEditorContainer.tsx

```diff
@@ -34,14 +46,23 @@ export function JsonEditorContainer(props: { json: any, templates: Template[] })
             console.log(err.toString())
         },
         onCreateMenu: (menuItems: MenuItem[], node: MenuItemNode) => {
-            const paths = node.paths[0];
+            // get current state
+            let currentJson = jsonEditorInstance.current?.getText();
+            if (currentJson) {
```

Comment:
Do not use the same variable for different purposes

Reply:
Done